index.html

```diff
@@ -4,8 +4,82 @@
      <meta charset="UTF-8">
      <meta name="viewport" content="width=device-width, initial-scale=1.0">
      <title>project 3</title>
+     <link rel="stylesheet" href="style.css">
+   <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Kanit:wght@300&display=swap" rel="stylesheet">
  </head>
  <body>
-     
+     <header>
+       <h1>
+        Design conference<br>
+         ——Typography In Web Design
+       </h1>
+      </header>
+       <nav>
+      <ul class="navigation" >  
+      <li><a href="#">Home</a></li>
+      <li><a href="#">Speakers</a></li>
+      <li><a href="#">Register</a></li>
+      <li><a href="#">Contact</a></li>
+      </ul> 
+      </nav>
+   <main>
+     <ul class="u-list-reset ">
+    <li class="talk1">
+     <img src="https://cdn.glitch.com/0e85a349-5917-49b5-9721-e8ec891917e8%2F1.png?v=1607879901347"
+        alt="Photo of Jordan Devos" />
+     <h3>Jordan Devos</h3>
+     <h4>Talk: Designing for Readability – A Guide to Web Typography<br>
+       Time：10 AM - 12 PM, February 1st, 2021
+     </h4>
+     <h6>
+       About the speaker: Jordan is an expert designer whose focus is on visual communications and whose work integrates strategy, service design, and UX/UI. She has worked in a range of companies, from corporate in-house to boutique design studios. A self-starter who wears many hats, Jordan is in her element leading a project or managing a team.
+     </h6>
```

Comment:
Same note about skipping heading levels. Also, this text more closely resembles a paragraph than a heading, do you agree?
```suggestion
     <p>
       About the speaker: Jordan is an expert designer whose focus is on visual communications and whose work integrates strategy, service design, and UX/UI. She has worked in a range of companies, from corporate in-house to boutique design studios. A self-starter who wears many hats, Jordan is in her element leading a project or managing a team.
     </p>
```